README.md

```diff
@@ -9,10 +11,9 @@ Endon is an Elixir library that provides helper functions for [Ecto](https://hex
 Yes, Ecto is great!  But there are a few things that are really annoying, and a little syntactic sugar can go a long way.  See the [Features](guides/features.md) page for more info.
 
 #### What does Endon mean?
-"Ecto" is a prefix from Greek έκτός (ektós) meaning "outside".  The opposite of Ecto is "Endon" (Greek ἔνδον) means "within, inner, absorbing, or containing".  Why the opposite of Ecto?  No good reason at all; naming is hard.
+"Ecto" is a prefix from Greek έκτός (ektós) meaning "outside".  The opposite of Ecto is "Endon" (Greek ἔνδον) means "within, inner, absorbing, or containing". Why the opposite of Ecto?  No good reason at all; naming is hard.
```

Comment:
I think this change is also superfluous - I used double spaces between sentences throughout so for consistency I think the double space here can be kept.

Reply:
I think this is the only outstanding issue - everything else looks good

Reply:
> I think this is the only outstanding issue - everything else looks good

Done.